modals/users.js

```diff
@@ -0,0 +1,23 @@
+const { v4 } = require('uuid');
+const md5 = require('md5');
+
+function createUser(userName, password, firstName, lastName, sessionId) {
```

Comment:
What `sessionId` is for?

Reply:
sessionId needed to store the sessions id, which will be needed for login session.

Reply:
You don't need to store `sessionId` in permanent storage